findRefOnGithub.test.js

```diff
@@ -89,7 +89,7 @@ test("fails to find ref (rate limiting)", () => {
   mockRefLookupFailure(action, "heads/master");
   mockCommitLookupRateLimit(action, "master");
   return expect(findRef(action)).rejects.toEqual(
-    `Unable to find SHA for nexmo/github-actions@master\nAPI rate limit exceeded for 1.2.3.4. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)`,
+    `Unable to find SHA for nexmo/github-actions@master\nAPI rate limit exceeded for 1.2.3.4. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.) Limit resets at: 4/9/2025, 8:08:44 AM`,
```

Comment:
```diff
    - API rate limit exceeded for 1.2.3.4. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.) Limit resets at: 4/9/2025, 8:08:44 AM
    + API rate limit exceeded for 1.2.3.4. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.) Limit resets at: 09/04/2025, 16:08:44
```

Being in the UK, I have a different locale format 😁 

Is there a way to mock the timezone here?

Reply:
Two ways we can probably go, we could in jest do something in the beforeEach() which seems silly to set the date that way, so less ideal probably is setting a const for the time to match the test behavior and doing the same .toLocaleString() call for it. 

I've done the latter for now as it was the simpler and less costly behavior, while setting time before each test isn't going to add much time, minimizing that behavior I thought would be better.